shfmt/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM ubuntu:bionic
+
+RUN \
+  apt-get update && \
+  apt-get install -y wget && \
+  rm -rf /var/lib/apt/lists/*
```

Comment:
is this rm needed?

Reply:
It's not necessary, but IIRC, it's a "best practice" to keep the resultant image smaller by removing cached artifacts.

Reply:
Okay, makes sense, found it here: https://docs.docker.com/develop/develop-images/dockerfile_best-practices/